styles/output/nursing-internal-pdf.css

```diff
@@ -1867,8 +1862,8 @@ h4.os-subtitle + .os-figure:not(.has-splash) > figure {
 }
 
 .os-table:not(.os-unstyled-container) > table > tbody > tr > td:not(:only-of-type) {
-  font-family: IBM Plex Sans, sans-serif;
-  border-left: 0.025rem solid black;
+  font-family: Mulish, sans-serif;
+  border-right: 0.025rem solid #000000;
```

Comment:
`border-left` has changed to `border-right`. same question^

Reply:
fixed mistake